edb/server/tenant.py

```diff
@@ -289,3 +289,19 @@ async def _reconnect_sys_pgcon(self):
             self._server.set_pg_unavailable_msg(None)
         finally:
             self._sys_pgcon_ready_evt.set()
+
+    def on_pgcon_broken(self, is_sys_pgcon=False):
```

Comment:
Missing typing is added in a future commit https://github.com/edgedb/edgedb/pull/5742/commits/ea82cdf68269ed2b0ca64aa394f1d719a07bb116